src/styles/grade.scss

```diff
@@ -0,0 +1,26 @@
+.grade {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 24px;
+  font-size: 10px;
```

Comment:
Font sizes should be defined in `rem` units instead of `px` to ensure better scalability and accessibility. Consider using the `rem` unit for font-size here. For example, if the root font-size is 16px, then 10px would be 0.625rem (`10px / 16px = 0.625rem`).